src/pages/events/dialogs/new.tsx

```diff
@@ -313,6 +350,63 @@ export const EventNewIncidentDialog: React.FC<EventNewIncidentDialogProps> = ({
               ))}
           </Select>
         </label>
+        {incident.skills ? (
+          <div className="flex gap-2 mt-2">
+            <Radio
+              name="skillsType"
+              label="Driver"
+              checked={incident.skills.skillsType === "driver"}
+              onChange={(e) =>
+                e.currentTarget.checked
+                  ? onChangeIncidentSkillsType("driver")
+                  : null
+              }
+            />
+            <Radio
+              name="skillsType"
+              label="Auto"
```

Comment:
Wanting to avoid the connonation that it's happening in the automous period. Technically the name for prog skills is "autonomous coding skills", and as much as I don't like that name, I think it's more clear to say "auto skills" as short for "auto coding skills"

Maybe Prog Skills would be better?